roles/extract_operator_bundle/tasks/extract_image_layer.yml

```diff
@@ -0,0 +1,34 @@
+---
+- debug:
+    var: image_layer
+
+- name: "Parse the digest of the image layer"
+  set_fact:
+    image_layer_digest: "{{ image_layer.digest.split(':')[1] }}"
+    image_layer_media_type: "{{ image_layer.mediaType }}"
+
+- name: "Check that the image image layer archive file exists"
+  stat:
+    path: "{{ operator_bundle_dir }}/{{ image_layer_digest }}"
+  register: layer_digest_stat_result
+
+- name: "Extract the layer into the operator test directory"
+  shell: "tar axf {{ operator_bundle_dir }}/{{ image_layer_digest }} -C {{ operator_work_dir }}"
```

Comment:
could you use `--auto-compress -xf`instead of `axf`?
We need't use long verion for everything but I have to check what is a purpose of "a" in man page :-)

Reply:
Yeah, that makes sense, changing.